doc/editor.rst

```diff
@@ -45,11 +30,9 @@ A "code cell" in Spyder is a block of lines, typically in a script, that can be
 This is much like a "cell" in MATLAB (except without any need to enable a "cell mode", since in Spyder, cells are detected automatically).
```

Comment:
The link to the IPython console page above this line is broken, i.e. it appears as `doc:ipythonconsole` in Netlify.

Unfortunately, I can't change it in a suggestion.

Reply:
Yup, its missing the leading `:` on doc. Not sure how that slipped through since it wasn't changed here and the build would fail if this was the case, preventing merging.

Reply:
Looking at the git blame, it was in fact my mistake from two years ago hehe